lib/scoreboard.rb

```diff
@@ -5,8 +5,11 @@ class Scoreboard
   FLPATH = File.join(File.dirname('../'), 'score.yml')
   Row = Struct.new(:name, :tries, :time, :timestamp)
   def initialize
-    @score = YAML.load_file(FLPATH) if File.exist?(FLPATH)
-    @score = [] unless @score
+    @score = if File.exist?(FLPATH)
+               YAML.load_file(FLPATH)
+             else
+               []
+             end
```

Comment:
`@score = File.exist?(FLPATH) && YAML.load_FILE(FLPATH) || []`

Just another way to express it.  Or you could also express it like this:

`@score = File.exist?(FLPATH) ? YAML.load_FILE(FLPATH) : []`